west.yml

```diff
@@ -72,7 +72,7 @@ manifest:
       revision: 30b7efa827b04d2e47840716b0372737fe7d6c92
     - name: nrfxlib
       path: nrfxlib
-      revision: 4de8eebe8cfecb4fd268868263eb6ede54bbb86b
+      revision: pull/112/head
```

Comment:
This PR does not need an updated west.yml

Reply:
@rugeGerritsen Are you saying the nrfxlib change is not needed? If it is needed then we need a PR for it to run CI. So we might as well include it here.